src/main/interface/cli.c

```diff
@@ -4253,6 +4273,9 @@ static void printConfig(char *cmdline, bool doDiff)
         cliPrintHashLine("name");
         printName(dumpMask, &pilotConfig_Copy);
 
+        cliPrintHashLine("display_name");
+        printDisplayName(dumpMask, &pilotConfig_Copy);
```

Comment:
Erg, this needs to be guarded by `#ifdef USE_OSD` as well.

Protip: When fixing problems like the other missing guard that you recently worked on, it is a good idea to, after the fix has been done, try and build one of the failing targets to make sure the fix is indeed correct - this also turns up repetitions of the same failure pattern that were hidden by the compiler erroring out after running into the first occurrence.